schemas/component-descriptions/README.md

```diff
@@ -266,27 +252,3 @@ descriptions to `${WORKSPACE}/install/<package_name>/component_descriptions`.
 install(DIRECTORY ./component_descriptions
         DESTINATION .)
 ```
-
-## CLion configuration for automatic validation
```

Comment:
Why not keep and update this for CLion and VSCode? Just change the link to the schema to our versioned one.

You can also just say something like: associate a component description with a JSON schema using the `$schema` property.

```json
{
  "$schema": "https://docs.aica.tech/schemas/1-0-0/component.schema.json"
}
```